### PR TITLE
Update Disk Store Usage Calculation While Registering Metrics

### PR DESCRIPTION
**Pull request by atris**

### CODE REVIEW

**Comment by atris** [comment]
cc @clebertsuconic @michaelandrepearce . This fixes the test failure post 2636

**Comment by clebertsuconic** [comment]
@atris did you actually run the test? it still failing after your fix.

you need to amend your commit description to include the JIRA ID as part of the message. Use the previous JIRA.

**Comment by clebertsuconic** [comment]
You fixed the calculation, but you didn't fix the test.

**Comment by atris** [comment]
Updated the test fix. The test suite and the erroring test pass now

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/plugin/MetricsPluginTest.java [hunk 1]
```diff
@@ -116,13 +116,14 @@ public int hashCode() {
...
+              new Metric("artemis.address.memory.usage", "Bytes used by all the addresses on broker for in-memory messages", 0.0),
               new Metric("artemis.connection.count", "Number of clients connected to this server", 1.0),
               new Metric("artemis.consumer.count", "number of consumers consuming messages from this queue", 0.0),
               new Metric("artemis.delivering.durable.message.count", "number of durable messages that this queue is currently delivering to its consumers", 0.0),
               new Metric("artemis.delivering.durable.persistent.size", "persistent size of durable messages that this queue is currently delivering to its consumers", 0.0),
               new Metric("artemis.delivering.message.count", "number of messages that this queue is currently delivering to its consumers", 0.0),
               new Metric("artemis.delivering.persistent_size", "persistent size of messages that this queue is currently delivering to its consumers", 0.0),
+              new Metric("artemis.disk.store.usage", "Memory used by the disk store", 0.0),
```
Comment (clebertsuconic) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/plugin/MetricsPluginTest.java:126]: I think this should be Bytes as well... can you change the description as well?

**Pull request closed without merging**